src/Reporter.cpp

```diff
@@ -594,7 +594,7 @@ int geopm_reporter_init(void)
     int err = geopm_time_string(NAME_MAX, start_time);
     if (!err) {
         try {
-            geopm::basic_reporter(start_time);
+            geopm::basic_reporter(start_time).init();
```

Comment:
AFAICT, init() was never called in the past (by the C++ runtime).  What are all the implications here?  I'm wondering what we should have seen not working properly before this change.

Reply:
This change should be reverted.  Thanks!